packages/ppx_heml/ppx_heml.0.1.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "PPX for deriving HTML templates"
+description: "PPX for deriving HTML templates. Includes %heml"
+maintainer: ["Petri-Johan Last"]
```

Comment:
Could you provide a contact for maintenance? An email address is what we usually use and is preferable, but it should be some way you can be contacted at least.